src/components/WorkOrder/WorkOrderView.js

```diff
@@ -33,9 +33,16 @@ const WorkOrderView = ({ workOrderReference }) => {
       setWorkOrder(null)
       setProperty(null)
       console.log('An error has occured:', e.response)
-      setError(
-        `Oops an error occurred with error status: ${e.response?.status}`
-      )
+
+      if (e.response?.status === 404) {
```

Comment:
This looks nice. I think we could also redirect them back to the home page right so they can perform another search? At the moment they would need to navigate back to the home page to do this if I understand correctly

Reply:
I'm thinking display the home page first and then show this message at the top